pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "pyansys"
+version = "2023.1.dev0"
+description = "Pythonic interfaces to Ansys products"
+readme = "README.rst"
+license = "MIT"
+homepage = "https://github.com/pyansys"
+repository = "https://github.com/pyansys/pyansys_landing"
+documentation = "https://docs.pyansys.com/"
+authors = [
+    "ANSYS, Inc. <pyansys.support@ansys.com>",
+]
+maintainers = [
+    "PyAnsys developers <pyansys.maintainers@ansys.com>",
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Information Analysis',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: MacOS',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+]
+
+packages = [
+    { include = "pyansys", from = "src" },
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4.0"
+importlib-metadata = {version = "^4.0", python = "<3.8"}
+ansys-mapdl-core = "==0.63.1"
+
+ansys-dpf-core = "==0.4.2"
+ansys-dpf-post = "==0.2.2"
+ansys-fluent-core = "==0.10.1"
+pyaedt = "==0.5.11"
+ansys-platform-instancemanagement = "==1.0.2"
+ansys-grantami-bomanalytics = "==1.0.1"
+ansys-openapi-common = "==1.1.1"
+
+# Optional dependencies
+ansys-mapdl-reader = {version = "==0.51.15", optional = true}
+# ansys-fluent-visualization = {version = "==0.4.0", optional = true}
+ansys-fluent-parametric = {version = "==0.4.1", optional = true}
+Sphinx = {version = "==5.1.1", optional = true}
+ansys-sphinx-theme = {version = "==0.4.2", optional = true}
+
+[tool.poetry.extras]
+mapdl-all = ["ansys-mapdl-reader"]
+# fluent-all = ["ansys-fluent-visualization","ansys-fluent-parametric",]
+fluent-all = ["ansys-fluent-parametric",]
+# all = ["ansys-mapdl-reader","ansys-fluent-visualization","ansys-fluent-parametric"]
+all = ["ansys-mapdl-reader","ansys-fluent-parametric"]

```

Comment:
I am not sure if it is possible to reference extra groups like this:

```toml
all = ["mapdl-all", "fluent-all"]
```

Reply:
Yeah, that'd be great. But nope 😆 not  possible. Already tried it in the commit history. I wish that could be possible.